src/Tools/dotnet-monitor/CollectionRules/Options/Triggers/AspNetRequestDurationOptions.cs

```diff
@@ -19,6 +19,7 @@ internal sealed class AspNetRequestDurationOptions :
             ResourceType = typeof(OptionsDisplayStrings),
             Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_AspNetRequestDurationOptions_RequestCount))]
         [Required]
+        [Range(1, int.MaxValue)]
         public int RequestCount { get; set; }
 
         [Display(
```

Comment:
I just noticed that we don't specify the range for `RequestDuration`; should we be? Just wanted to double check on this before the PR goes in since in `AspNetRequestDurationTriggerSettings` in `EventPipe` we specify a min of `00:00:00` and a max of `01:00:00`